src/@types/types.d.ts

```diff
@@ -158,6 +160,12 @@ export type ChromeModule = {
   modules?: RemoteModule[];
   defaultDocumentTitle?: string;
   fullProfile?: boolean;
+  props?: ExtraProps | any;
```

Comment:
You should utilize the config field. We agreed with frontend operator on sending any additional data trough that field, so we don't have to add arbitrary fields.